reconcile/queries.py

```diff
@@ -3069,3 +3072,11 @@ def get_app_metadata(app_path: str) -> dict:
 def get_blackbox_exporter_monitoring_provider() -> dict:
     gqlapi = gql.get_api()
     return gqlapi.query(BLACKBOX_EXPORTER_MONITORING_PROVIDER)["providers"]
+
+
+def get_smtp_client_settings() -> SmtpSettingsV1:
+    if _settings := smtp_client_config_query(query_func=gql.get_api().query).settings:
+        if not _settings[0].smtp:
+            raise Exception("settings.smtp missing")
```

Comment:
Should have some general missing configuration exception?

Reply:
🤦 I wanted to change before pushing it.